readthedocs/templates/builds/build_detail.html

```diff
@@ -174,6 +174,14 @@
             {% endblocktrans %}
           </p>
         </div>
+      {% elif build.finished and not build.deprecated_config_used  %}
+        <div class="build-ideas">
+          <p>
+            {% blocktrans trimmed %}
+              <strong>Your build is properly using a configuration file ✅</strong><br/>
```

Comment:
I thought we said the `cat` command was enough and we didn't want this block here. Am I right?

Reply:
The notification block is minor, but it also feels like a one-off addition to the UI. I'd say the cat command is okay for now.

But if we really want a positive message to the user here, I'm mostly impartial to this block _on a temporary basis_. Long term, I don't want to be stubbing in UI like this though, but we can always remove this after the deprecation as well -- when the config become mandatory.

Reply:
I removed it for now, don't need to block this on UI 👍